lms/djangoapps/student_account/views.py

```diff
@@ -396,4 +397,11 @@ def account_settings_context(request):
             'disconnect_url': pipeline.get_disconnect_url(state.provider.NAME),
         } for state in auth_states]
 
+    external_auth_map = None
+    try:
+        external_auth_map = ExternalAuthMap.objects.get(user=user)
+    except:
+        pass
```

Comment:
Nit: You could save a line by just assigning `external_auth_map = None` in the `except` block, instead of simply `pass`ing.